app/bundles/CoreBundle/Twig/Helper/SecurityHelper.php

```diff
@@ -28,6 +28,16 @@ public function getName(): string
         return 'security';
     }
 
+    /**
+     * Helper function to check if user is an Admin.
+     *
+     * @return bool
+     */
+    public function isAdmin()
```

Comment:
We don't need the return annotation and specify the return type

Reply:
I am unsure If I can add a strict return type for the public method. @escopecz Should I?

Reply:
If it's a new method, then yes. Or is there some reason why you thing it could break BC?